api/all.yaml

```diff
@@ -2278,6 +2375,138 @@ components:
+        format_options:
+          $ref: '#/components/schemas/OptionsKVPairs'
+        schema:
+          $ref: '#/components/schemas/ColumnInfos'
+        partition_columns:
+          type: array
+          items:
+            type: string
+          description: The partition columns of the table (should be set if changed).
+        properties:
+          $ref: '#/components/schemas/SecurablePropertiesMap'
+        created_time:
+          type: string
+          description: The time this metadata action was created (this is generated by the Delta client writing to the table).
+      required:
+        - delta_table_id
+    Protocol:
+      type: object
+      description: Represents a Delta protocol action (see https://github.com/delta-io/delta/blob/master/PROTOCOL.md#protocol-evolution for more information).
+      properties:
+        min_reader_version:
+          type: integer
+          format: int32
+          description: The minimum reader version defined by the protocol.
+        min_writer_version:
+          type: integer
+          format: int32
+          description: The minimum writer version defined by the protocol.
+        reader_features:
+          type: array
+          items:
+            type: string
+          description: The reader features defined by the protocol (can be empty if the table has no reader features set).
+        writer_features:
+          type: array
+          items:
+            type: string
+          description: The writer features defined by the protocol (can be empty if the table has no writer features set).
+      required:
+        - min_reader_version
+        - min_writer_version
+    OptionsKVPairs:
+      type: object
+      description: A map of key-value options attached to the entity.
+    ColumnInfos:
+      type: object
+      properties:
+        columns:
+          type: array
+          items:
+            $ref: '#/components/schemas/ColumnInfo'
+          maxItems: 32768
+          description: The array of column descriptions.
+    CommitInfo:
+      type: object
+      properties:
+        version:
+          type: integer
+          format: int64
+          description: The version of this commit.
+        timestamp:
+          type: integer
+          format: int64
+          description: The timestamp for when the commit was made.
+        file_name:
+          type: string
+          description: The filename of the UUID-based commit file.
+        file_size:
+          type: integer
+          format: int64
+          description: The size of the commit file in bytes.
+        file_modification_timestamp:
+          type: integer
+          format: int64
+          description: The modification time of the commit file.
+        is_disown_commit:
+          type: boolean
+          description: Indicates if this commit is a disown commit that removes UC as the commit coordinator from the table.
+    CommitResponse:
+      type: object
```

Comment:
Are we keeping CommitResponse empty?
We can add CommitInfo & latest_backfilled_version

Reply:
Let's keep it empty actually.